### DISCUSSION TITLE
Can't open nexreder CLI

### DISCUSSION THREAD

**Discussion by Carrabba** [inlife/nexrender#853] [Q&A]
Every time i try to start the nexrender cli it crashes even if i try to log in as administrator.
What can i do?

**Comment by inlife** [comment]
Please show screenshots or videos, and provide crash report if possible

**Comment by Carrabba** [comment]
I tried to open the nexrender CLI both as a normal user and as an administrator but in both cases it closes by itself.

Even using windows 11 I have the same problem


https://user-images.githubusercontent.com/104204039/220861584-b8f61d46-e71e-45d8-b1ea-7fb77f99fced.mp4

**Reply by inlife** [reply]
@Carrabba To clarify, it is a CLI application. These kinds of applications are always launched via a command line.

To do so:
1. Open cmd.exe or PowerShell
2. Change path to the location of the nexrender executable via `cd`
3. Run nexrender executable via console
